### PR TITLE
Hide wayfinding sub paragraphs in 1004.2

### PR DESCRIPTION
**Pull request by ascott1**
This splits up the sub paragraphs in the wayfinding and hides them in section 1004.2. We also have the ability to easily add this feature to other sections if/when needed.

## Screenshots

screen shot 2015-11-19 at 3 39 20 pm

screen shot 2015-11-19 at 3 39 28 pm

## Review

@willbarton or @grapesmoker

### CODE REVIEW

File: regulations/static/regulations/css/less/module/header.less [hunk 1]
```diff
@@ -408,3 +408,20 @@ Small Screens
...
+
+
+/*
+Wayfinding Overrides
+=====================
+In CFPB regs we sometimes have markerless paragraphs
+The markup contains these ID's so that we can link to them
+but we want to hide them in the wayfinding header.
+*/
+
```
Comment (willbarton) [inline regulations/static/regulations/css/less/module/header.less]: Flagging this for future... but I can't help thinking there's gotta be a better place to do this.

**Comment by willbarton** [comment]
:+1:

File: regulations/static/regulations/css/less/module/header.less (hunk 1 above)
Comment (ascott1) [inline regulations/static/regulations/css/less/module/header.less]: Yeah, maybe a cfpb-overrides file?

Comment (willbarton) [inline regulations/static/regulations/css/less/module/header.less]: Yeah... well, I think it's part of the larger conversation about theming the front-end.

**Comment by cmc333333** [comment]
We took this approach, effectively teaching the JS about paragraphs which don't have markers: https://github.com/18F/regulations-site/pull/20/ 

Related - the search templates probably also need to be aware: https://github.com/18F/regulations-site/pull/18

**Comment by willbarton** [comment]
@cmc333333 I feel like both approaches are stop-gaps to a certain degree. With the https://github.com/cfpb/regulations-schema we'll be able to be explicitly specify markers, and then we can take that into account on the front-end. We've also had some discussion about what should actually be shown for markerless paragraphs... and that's something we want to explore a bit more with our SMEs and non-existent UXers.

**Comment by cmc333333** [comment]
@willbarton maybe a transition plan would be to pass that information through to the template the way we'd expect it to come from the schema? Then the frontend can start solving the general problem independently of when the schema is finished.

**Pull request merged**